HW3.3.java

```diff
@@ -1 +1,128 @@
+package com.company;
 
+public enum Direction {
```

Comment:
are you moved Direction into separate file?
or you moved it on the start of the class Robot?

Reply:
I had already moved it to a separate file